src/config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "PagerDuty-Trigger",
+  "type": "trigger",
+  "execProgram": "node",
+  "main": "app.js",
+  "imgUrl": "pd.png",
+  "version": "1.0.0",
+  "description": "PagerDuty Trigger",
+  "methods": [
+    {
+      "name": "INCIDENT_WEBHOOK",
+      "route": "post /webhook/pagerduty/incident",
+      "viewName": "Find user on call",
```

Comment:
@ilanyaniv I think this should be `Find user on call`, but instead `PageDuty Event` because you are asking for an event parameter